chore: set `CONTRACTS_NODE_PATH` env variable for e2e tests (#209)
r0gue-io/pop-cli: README.md

For end-to-end testing you will need to have a Substrate node with `pallet contracts`.
You do not need to run it in the background since the node is started for each test independently.
To install the latest version:
```
cargo install contracts-node --git https://github.com/paritytech/substrate-contracts-node.git
```

Run e2e testing on the Smart Contract:

```sh

> # Run e2e tests for an existing smart contract

Reviewer: Don't see the use for this comment, more cases like this in the README. Most of them have a description already above the code import. Should be fixed when fixing #183

 pop test contract  -p ./my_contract --features e2e-tests
```

> If you want to run any other node with `pallet-contracts` you need to change `CONTRACTS_NODE` environment variable:

Reviewer: ```suggestion
If you want to run a different node with `pallet-contracts` you need to change `CONTRACTS_NODE` environment variable:
```

```sh
export CONTRACTS_NODE="YOUR_CONTRACTS_NODE_PATH"
```

> You can automate this process with the `--contracts-node` flag:

Reviewer: ```suggestion
Or, use the `--contracts-node` flag:
```

This should be the first choice TMO, second doing the export

Author: You are right, actually the export CONTRACTS_NODE is not needed anymore

For end-to-end testing you will need to have a Substrate node with `pallet contracts`.
You do not need to run it in the background since the node is started for each test independently.
To install the latest version:
```
cargo install contracts-node --git https://github.com/paritytech/substrate-contracts-node.git
```

Run e2e testing on the Smart Contract:

```sh
# Run e2e tests for an existing smart contract
 pop test contract  -p ./my_contract --features e2e-tests
```

> If you want to run a different node with `pallet-contracts` you need to change `CONTRACTS_NODE` environment variable using the  `--contracts-node` flag:

Reviewer 2: Shouldnt this be `--node` flag?

Author: Yes! Apologies for this mistake
